python/requirements.txt

```diff
@@ -1,14 +1,17 @@
-numpy==1.14.5
```

Comment:
Why have we removed versions

Reply:
As discussed it was done to sync versions across setup.py and requirements.txt - I created #767 for this one